dune-project

```diff
@@ -1 +1,16 @@
-(lang dune 1.0)
+(lang dune 2.8)
+(name io_uring)
+(license BSD-3-Clause)
+(generate_opam_files true)
+(authors "")
+(source (github mt-caret/io_uring))
+(package
+  (name io_uring)
+  (synopsis "IO uring bindings for ocaml")
+  (authors "mt-caret")
+  (maintainers "mt-caret <mtakeda.enigsol@gmail.com>")
+  (homepage "https://github.com/mt-caret/io_uring")
+  (bug_reports "https://github.com/mt-caret/io_uring")
+  (version 1.0)
+  (depends (core (>= "v0.14.1")) (ctypes (>= "0.17.1")) (ctypes-foreign (>= "0.18.0")) (ocaml (>= "4.08.0")))
```

Comment:
I'm kind of surprised that we're including ctypes here. Are you planning to depend on ctypes at some point in this PR?

Reply:
Ah a mistake copied in from a prototype